src/ziggurat/server/routes.clj

```diff
@@ -22,11 +24,15 @@
    ["v1/dead_set" {:get (ds/get-view)}]
    [true (fn [_req] (ring.util.response/not-found ""))]])
 
+(defn- swagger-enabled? [] (get-in-config [:swagger :enabled]))
+
 (defn handler [actor-routes]
   (-> routes-prefix
       (conj (vec (concat actor-routes (get-routes))))
       (bidi/make-handler)
       (m/wrap-hyphenate)
+      (cond->
```

Comment:
why is a threaded cond used here? I don't think it is necessary

Reply:
also, can we have a custom middleware here that wraps this logic?

Reply:
Done.. wrapped it in custom middleware